src/data/donations.js

```diff
@@ -13,10 +13,21 @@ async function createDonation(ideaId, userId, sum, firstName) {
             },
             body: JSON.stringify(body),
         });
+        console.log(response);
```

Comment:
console.log

Reply:
Fixed. And I'm surprised, that that's it! ))) I thought there will be tons of comments ))